credits/credits_source.tex

```diff
@@ -30,9 +30,9 @@
 
 \noindent Thanks to Josh Zucker who wrote the first draft of several chapters (particularly The Geometry of Complex Numbers, Vitamin $i$, Matrix Multiplication, and Inverses part I).
 
-Also, thanks to my fellow teachers at Gunn High School Dave Deggeller and Danny Hahn for resurrecting the use of this pamphlet in their 11\textsuperscript{th} grade Analysis classes and to their Analysis classes of 2013-14 who took on the project of retyping the text after the loss of the soft copy in a computer theft.
+Also, thanks to my fellow teachers at Gunn High School Dave Deggeller and Danny Hahn for resurrecting the use of this pamphlet in their 11\textsuperscript{th} grade Analysis classes and to their Analysis classes of 2013--14 who took on the project of retyping the text after the loss of the soft copy in a computer theft.
```

Comment:
En dash is correct

Reply:
I'm aware, I mean a more general one
I don't know what is correct in general because I am non english pro
Gamepro may be helpful

Reply:
Hyphens for combining words, en dashes for ranges, em dashes for separating phrases

The cow-eating monster (estimated to be 3–4 km tall) stepped on the worst building it could've stepped on—our food storage facility.

Reply:
Thank you!!!!!!

Reply:
yeet

Reply:
would an en dash surrounded by spaces will work though? or is that not super correct

Reply:
idk about spaces oof

Reply:
![image](https://user-images.githubusercontent.com/22133785/62012167-7a9df280-b137-11e9-8eef-58551b5bf87f.png)

Reply:
but this is annoying because Google Docs,

1. doesn't do `--` -> `—` unless you put a space after it

2. doesn't break words around dashes when wrapping